webpack.config.js

```diff
@@ -4,10 +4,18 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const webpack = require('webpack');
 
 module.exports = function (env) {
+    
     const isProd = env === "prod";
     if (isProd) {
         process.env.NODE_ENV = "production";
     }
+
+    console.info(`This build is in production mode: ${isProd}`);
+
+    if(process.env.OPENWEATHER_API_KEY === undefined) {
```

Comment:
It's better to replace this check with something not so strict `!process.env.OPENWEATHER_API_KEY` to check for null and empty string as well